v63003/u_inref/gtm8779.csh

```diff
@@ -0,0 +1,44 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Testing for DBFREEZEON/DBFREEZEOFF message when freeze state is changed
+#
+
+echo "# Setting Default Region to FROZEN"
+$gtm_tst/com/dbcreate.csh mumps 1 >>& create.out
+
+# Setting freeze to off initially so we can ensure the first command
+# is changing the freeze status
+$MUPIP FREEZE -OFF DEFAULT >>& init.out
+
+set t1 = `date +"%b %e %H:%M:%S"`
+# Sleep commands to ensure a difference in time of at least 1 second so getoper.csh
+# will catch the message every time
+sleep 1
+$MUPIP FREEZE -ON DEFAULT
+set t2 = `date +"%b %e %H:%M:%S"`
+sleep 1
+echo "# Verifying System received a DBFREEZEON message"
+$gtm_tst/com/getoper.csh "$t1" "$t2" t1t2.txt
+
+cat t1t2.txt |& $grep "DBFREEZEON" | $tst_awk '{print $6}'
+
+echo "# Setting Default Region to UNFROZEN"
+$MUPIP FREEZE -OFF DEFAULT
+set t3 = `date +"%b %e %H:%M:%S"`
+echo "# Verifying System received a DBFREEZEOFF message"
+$gtm_tst/com/getoper.csh "$t2" "$t3" t2t3.txt
+
+cat t2t3.txt |& $grep "DBFREEZEOFF" | $tst_awk '{print $6}'
+
+$gtm_tst/com/dbcheck.csh >>& check.out
```

Comment:
Instead of the above, I would like to have 3 regions AREG, BREG and DEFAULT. And run the below two test scenarios.

1) Have AREG freeze set to off, BREG freeze set to ON and DEFAULT freeze set to off. And do a MUPIP FREEZE -ON "*" to verify that DBFREEZEON messages get sent only for AREG and DEFAULT and that no DBFREEZEOFF messages get sent.

2) Have AREG freeze set to off, BREG freeze set to ON and DEFAULT freeze set to on. And do a MUPIP FREEZE -OFF "*" and verify DBFREEZEOFF only gets sent for BREG and DEFAULT and no DBFREEZEON messages get sent.